packages/react-native/React/CoreModules/RCTDevLoadingView.mm

```diff
@@ -112,38 +114,53 @@ - (void)showMessage:(NSString *)message color:(UIColor *)color backgroundColor:(
   }
 
   dispatch_async(dispatch_get_main_queue(), ^{
+    if (RCTRunningInTestEnvironment()) {
+      return;
+    }
+
     self->_showDate = [NSDate date];
 
-    if (!self->_window && !RCTRunningInTestEnvironment()) {
```

Comment:
I removed this check because we teardown the window every time inside of `hide` method anyway.